### ISSUE TITLE
List of Enum values fails

### ISSUE THREAD

**Issue by hippysurfer** [jsonpickle/jsonpickle#165]
Hi

This might be a duplicate of issue #140 , if so I apologies, but this is a simpler test case.

It looks like something is going wrong but I can't pin down whether it is the encoder or decoder. I have spent a few hours trying to trace this through but I have not come up with a fix.

When the JSON is decoded, the second element of the list should be a reference to the C.VALUE object, but instead it is a raw list [1,2,3].

Here is a minimal test case.

This is a simple workaround, which is to use a tuple for the VALUE list rather than a list but I thought it worth reporting the issue.

```
from enum import Enum
from jsonpickle import encode, decode


class C(Enum):
    VALUE = [1, 2, 3]


my_list = [C.VALUE, C.VALUE]

assert decode(encode(my_list))[1] == my_list[1]
```

Best regards and thank you for jsonpickle - it is fantastically useful.

Richard

**Comment by ZENALC** [comment]
This is no longer an issue. Upon running the code above, the assertion passes.

**Comment by davvid** [comment]
Thanks for the report.  This was likley fixed in v1.4.
